packages/web-components/fast-components-msft/docs/api-report.md

```diff
@@ -0,0 +1,257 @@
+## API Report File for "@microsoft/fast-components-msft"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { Anchor } from '@microsoft/fast-foundation';
+import { Badge } from '@microsoft/fast-foundation';
+import { BaseProgress } from '@microsoft/fast-foundation';
+import { Button } from '@microsoft/fast-foundation';
+import { Card } from '@microsoft/fast-foundation';
+import { Checkbox } from '@microsoft/fast-foundation';
+import { DensityOffset } from '@microsoft/fast-components-styles-msft';
+import { DesignSystem } from '@microsoft/fast-components-styles-msft';
+import { DesignSystemProvider } from '@microsoft/fast-foundation';
+import { Dialog } from '@microsoft/fast-foundation';
+import { Divider } from '@microsoft/fast-foundation';
+import { Flipper } from '@microsoft/fast-foundation';
+import { Radio } from '@microsoft/fast-foundation';
+import { RadioGroup } from '@microsoft/fast-foundation';
+import { Slider } from '@microsoft/fast-foundation';
+import { SliderLabel } from '@microsoft/fast-foundation';
+import { Switch } from '@microsoft/fast-foundation';
+import { Tab } from '@microsoft/fast-foundation';
+import { TabPanel } from '@microsoft/fast-foundation';
+import { Tabs } from '@microsoft/fast-foundation';
+import { TextArea } from '@microsoft/fast-foundation';
+import { TextField } from '@microsoft/fast-foundation';
+
+// @public (undocumented)
+export type BadgeAppearance = "accent" | "lightweight" | "neutral" | string;
+
+// @public (undocumented)
+export class FASTAnchor extends Anchor {
+}
+
+// @public (undocumented)
+export class FASTBadge extends Badge {
+    // (undocumented)
+    appearance: BadgeAppearance;
+    }
+
+// @public (undocumented)
+export class FASTButton extends Button {
+}
+
+// @public (undocumented)
+export class FASTCard extends Card {
+}
+
+// @public (undocumented)
+export class FASTCheckbox extends Checkbox {
+}
+
+// @public (undocumented)
+export class FASTDesignSystemProvider extends DesignSystemProvider implements Omit<DesignSystem, "contrast" | "direction" | "fontWeight" | "neutralForegroundDarkIndex" | "neutralForegroundLightIndex"> {
+    // (undocumented)
+    accentBaseColor: string;
```

Comment:
@nicholasrice @EisenbergeEffect should we take a task to add typedoc comments to some of these?

Reply:
I think so. I plan to add a bunch to fast-element while I'm traveling on Tuesday. One of the benefits of the api report is that it will show us where we're missing docs. It also helps us to see when we make breaking changes to APIs. One thing the api-extractor team recommends is to actually use code-owners on the api-report file so that you can ensure that specific people review PRs that affect that file, since it represents changes to docs or public APIs. Seems like an interesting idea.